docroot/modules/custom/civictheme_migrate/src/Utils/MediaHelper.php

```diff
@@ -205,22 +205,10 @@ public static function lookupFileFromUri(string $uri): ?FileInterface {
     }
 
     // Get the scheme from the file URI.
-    $uriParts = explode('://', $uri);
```

Comment:
Had a different thought when added. File names can allow us to query file records.